config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
+
+  resources :employee
+  resources :assignment
+  resources :week
```

Comment:
I prefer to use `only: [:index, :new, :create]` or whatever actions if I'm ever not using all of the CRUD actions.